safe-global/safe-wallet-web
# feat: Import and export counterfactual safes

## What it solves

Resolves #3959 

## How this PR fixes it

- Adds the undeployedSafesSlice to the import and export page in settings

## How to test it

1. Create a counterfactual safe
2. Go to the Settings > Data
3. Export your Data
4. Observe it displays that an undeployed safe will be exported
5. Open a new Browser or clear your local storage
6. Go to the Settings > Data page
7. Import the file
8. Observe it displays that an undeployed safe will be imported
9. Observe the safe appearing in the sidebar
10. Observe no errors when opening the safe

## Screenshots
<img width="836" alt="Screenshot 2024-07-22 at 11 11 52" src="https://github.com/user-attachments/assets/5be40981-41aa-4032-85b1-4de00e0a997f">

## Checklist
